DatadogInternal/Sources/Context/DatadogContext.swift

```diff
@@ -186,9 +186,15 @@ public protocol AdditionalContext {
 extension DatadogContext {
     /// Gets an additional context value.
     ///
-    /// - Parameter type: The value type.
+    /// - Parameter type: The additional context type.
     /// - Returns: The `Context` if found
     public func additionalContext<Context>(ofType type: Context.Type = Context.self) -> Context? where Context: AdditionalContext {
         additionalContext[type.key] as? Context
     }
+
+    /// Gets an additional context value.
+    /// - Parameter context: The additional context.
+    public mutating func set<Context>(additionalContext context: Context?) where Context: AdditionalContext {
```

Comment:
question/ How can `additionalContext` be removed after it was previously set? 🤔 💭

Reply:
By using this [method](https://github.com/DataDog/dd-sdk-ios/blob/develop/DatadogInternal/Sources/DatadogCoreProtocol.swift#L198), I can also add a method to the context itself 👍